providers/common/der/der_rsa.h.in

```diff
@@ -23,6 +23,9 @@ int ossl_DER_w_RSASSA_PSS_params(WPACKET *pkt, int tag,
                                  const RSA_PSS_PARAMS_30 *pss);
 /* Subject Public Key Info */
 int ossl_DER_w_algorithmIdentifier_RSA(WPACKET *pkt, int tag, RSA *rsa);
+int ossl_DER_w_algorithmIdentifier_RSA_params(WPACKET *pkt, int tag,
```

Comment:
This should really be named something like `ossl_DER_w_algorithmIdentifier_RSA_PSS()`.  At some point, we'll have the same thing for RSA-OAEP, which also comes with params...

Reply:
Renamed